components/esp_modem/examples/modem_tcp_client/main/modem_client.cpp

```diff
@@ -23,14 +23,25 @@
 #include "tcp_transport_mbedtls.h"
 #include "tcp_transport_at.h"
 
-#define BROKER_URL "test.mosquitto.org"
+#define USE_TLS 0
```

Comment:
This defines could be simplified by having the URL define directly without composing the macros. Also is the `USE_TLS` define a left over?